### DISCUSSION TITLE
Branch Push Access

### DISCUSSION THREAD

**Discussion by mishukdutta-toast** [OpenAPITools/openapi-diff#349] [Q&A] [answered]
Hi there, i've been trying to push branch with fix for https://github.com/OpenAPITools/openapi-diff/issues/348
But I keep getting `remote: Permission to OpenAPITools/openapi-diff.git denied to mishukdutta-toast.`
Is there a place where I'll have to request write access to the repo?

**Comment by joschi** [comment, marked as answer]
@mishukdutta-toast You need to fork this repository into your own account.

There you can create a branch with your changes and open a pull request on this repository for review and feedback, and eventually we'll hopefully merge your changes. 😃 

See also https://docs.github.com/en/get-started/quickstart/github-flow.
